web/backend/src/controllers/dela.ts

```diff
@@ -263,6 +238,30 @@ delaRouter.use('/*', (req, res) => {
   }
 
   const bodyData = req.body;
+
+  // special case for voting
+  const match = req.baseUrl.match('/api/evoting/forms/(.*)/vote');
```

Comment:
why not set this up as a specific endpoint ? It would be more idiomatic, unless I'm missing something.

Reply:
That's what we did, but then `sendToDela` fails, because it uses `baseUrl`, which is different in `delaRouter.Post` and `delaRouter.use`.